pkg/operator/status/condition.go

```diff
@@ -38,7 +40,7 @@ func unionCondition(conditionType string, defaultConditionState operatorv1.Condi
 		return OperatorConditionToClusterOperatorCondition(unionedCondition)
 	}
 
-	if len(badConditions) == 0 {
+	if len(badConditions) == 0 || (gated && earliestTransitionTime(badConditions).Add(time.Minute).After(time.Now())) {
```

Comment:
you need to assign these things to bools with names so we can read this logic.  This is inscrutable.

Reply:
> you need to assign these things to bools with names so we can read this logic. This is inscrutable.

but I think it's right

Reply:
Done